### PR TITLE
[Azure Stack] Handle DNS Double Async Headers

### PR DESCRIPTION
**Pull request by deathly809**
## Description

The DNS Module does not handle having both the Location and Async header.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by deathly809** [comment]
Build/Published/Verified locally.

File: src/StackAdmin/Dns/Commands.Dns/Models/DoubleFetchHandler.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+using System;
```
Comment (cormacpayne) [inline src/StackAdmin/Dns/Commands.Dns/Models/DoubleFetchHandler.cs]: @deathly809 please add the license header

Comment (bganapa) [reply src/StackAdmin/Dns/Commands.Dns/Models/DoubleFetchHandler.cs]: The file has headers

**Comment by bganapa** [comment]
@deathly809  This PR should have gone in to StackAdmin-1.5.0 branch

**Comment by bganapa** [comment]
Updating my other PR https://github.com/Azure/azure-powershell/pull/7228 with the commits from this one

File: src/StackAdmin/Dns/Commands.Dns/Models/DoubleFetchHandler.cs (hunk 1 above)
Comment (deathly809) [reply src/StackAdmin/Dns/Commands.Dns/Models/DoubleFetchHandler.cs]: No it doesn't... lol.  Let me do a PR to add missing headers.

**Pull request merged**